main.go

```diff
@@ -320,6 +298,11 @@ func main() {
 			kprobes = append(kprobes, kp)
 		}
 	}
+	if !useKprobeMulti {
```

Comment:
Nit: we should probably move the `link.KprobeMulti()` invocation from above to bellow.